### DISCUSSION TITLE
Multiple open source licenses?

### DISCUSSION THREAD

**Discussion by callumlocke** [zed-industries/zed#8011] [Feature Requests]
The repo has 3 different LICENSE files (AGPL, Apache, and GPL). I've never seen that before. Does it mean people are free to choose which single license they want to be bound by, or that they must comply with all three at once (I'm not sure if that's possible), or something else?

**Comment by SomeoneToIgnore** [comment]
Something else: parts of the repo are licensed differently and you have to refer to the corresponding module's license, see `license` field in each crate's `Cargo.toml`

In particular:
* `collab`-related cloud services are AGPL-licensed, e.g. https://github.com/zed-industries/zed/blob/fe8cc2c825734f7e8d5852cf3ab275809646e490/crates/collab/Cargo.toml#L8
* `gpui` framework and small utils crates are Apache-licensed: https://github.com/zed-industries/zed/blob/fe8cc2c825734f7e8d5852cf3ab275809646e490/crates/gpui/Cargo.toml#L8
* the rest of the code is GPL-licensed, e.g. Zed itself: https://github.com/zed-industries/zed/blob/fe8cc2c825734f7e8d5852cf3ab275809646e490/crates/zed/Cargo.toml#L7

**Reply by uncenter** [reply]
What license would the logo be under? I'm guessing `GPL-3.0-or-later`?
